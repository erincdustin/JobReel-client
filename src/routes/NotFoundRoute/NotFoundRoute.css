@keyframes bounce {
  0% {
    transform: translateY(-100%);
    opacity: 1;
  }
  5% {
    transform:translateY(-100%);
    opacity: 1;
  }

  15% {
    transform:translateY(0);
    padding-bottom: 5px;
  }

  30% {
    transform:translateY(-50%);
  }

  40% {
    transform:translateY(0%);
    padding-bottom: 6px;
  }

  50% {
    transform:translateY(-30%);
  }

  70% {
      transform:translateY(0%);
      padding-bottom: 7px;
  }
  80% {
    transform:translateY(-15%);
  }
  90% {
    transform:translateY(0%);
    padding-bottom: 8px;
  }
  95% {
    transform:translateY(-7%);
  }
  97% {
    transform:translateY(0%);
    padding-bottom: 9px;
  }
  99% {
    transform:translateY(-3%);
  }
  100% {
    transform:translateY(0);
    padding-bottom: 9px;
    opacity: 1;
  }
}

.wrapper {
  position: relative;
  background-image: url("../../assests/notFoundBackground.jpg");
  height: 100%;
  min-height: 450px;
  width: 100%;
  margin: auto;
  background-size: cover;
  background-repeat: no-repeat;
  border-radius: 2em;

  grid-column-start: 1;
  grid-column-end: 9;
}

.description-title {
  margin: auto;
  text-align: center;
  border-radius: 10px 44px 79px 63px;
  border: 1px solid #ececec;
  width: 30%;
  background-color: rgb(35, 20, 78);

}
.description-title > h3 {
  margin-top: 40%;
  color: #ececec;
}

.description {
  text-align: center;
  margin-top: 10%;
  width: 20%;
  min-width: 250px;
  margin-left:auto;
  margin-right: auto;
  z-index: 2;
}

.description > p, h4 {
  color: #ececec
}

.backToDash:hover {
  cursor: pointer;
  transform: scale(1.2);
  transition: all 0.2s ease-in-out;
}

#backToDash > img {
  width: 50px;
  height: 50px;
}
.astro {
  width: 100px;
  height: 100px;
  color: #ececec;
  position: relative;
  transform: rotate(45);
  float: right;
  margin-right: 10%;
  z-index: 1;
  animation: bounce 5s;
  animation-iteration-count: infinite;
}

.credit {
  display: none;
}