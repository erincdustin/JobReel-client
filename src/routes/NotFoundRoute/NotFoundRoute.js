import React from 'react';
import './NotFoundRoute.css';
import astronaut from '../../assests/astronaut-ingravity.svg';
import { Link } from 'react-router-dom'
import logo from '../../assests/jobreellogo200.png'

export default class NotFoundRoute extends React.Component{

  render(){
    return(
      <div>
      <section className="notfound">
        <div className='wrapper'>
          <div className='description-title'>
            <h4>Wooow there, Astronaut.
              <br/>
              Stop right there</h4>
          </div>
          <div className='description'>
            <h4>Welcome to the <em>OUTERSPACE</em></h4>
            <p>We are glad to see you here but you still need to find a job, dont you?</p>
            <div className='backToDash'>
              <Link id='backToDash'to='/dashboard'>
                <img src={logo} alt='jobreel-logo'/>
              </Link>
            </div>
          </div>
          <div className='astro'>
            <img src={astronaut} alt='astronaut'/>
          </div>
        </div>
      </section>
      <div className='credit'>Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/"             title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"             title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
      </div>
      
    )
  }
}