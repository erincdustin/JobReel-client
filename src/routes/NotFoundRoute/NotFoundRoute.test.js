import React from 'react'
import ReactDOM from 'react-dom'
import NotFoundRoute from './NotFoundRoute'

describe('NotFoundRoute smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<NotFoundRoute />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})