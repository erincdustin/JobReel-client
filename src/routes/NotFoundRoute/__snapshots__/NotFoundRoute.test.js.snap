// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundRoute snapshot test renders the UI as expected 1`] = `
<div>
  <section
    className="notfound"
  >
    <div
      className="wrapper"
    >
      <div
        className="description-title"
      >
        <h4>
          Wooow there, Astronaut.
          <br />
          Stop right there
        </h4>
      </div>
      <div
        className="description"
      >
        <h4>
          Welcome to the 
          <em>
            OUTERSPACE
          </em>
        </h4>
        <p>
          We are glad to see you here but you still need to find a job, dont you?
        </p>
        <div
          className="backToDash"
        >
          <Link
            id="backToDash"
            to="/dashboard"
          >
            <img
              alt="jobreel-logo"
              src="jobreellogo200.png"
            />
          </Link>
        </div>
      </div>
      <div
        className="astro"
      >
        <img
          alt="astronaut"
          src="astronaut-ingravity.svg"
        />
      </div>
    </div>
  </section>
  <div
    className="credit"
  >
    Icons made by 
    <a
      href="https://www.flaticon.com/authors/freepik"
      title="Freepik"
    >
      Freepik
    </a>
     from 
    <a
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
      www.flaticon.com
    </a>
     is licensed by 
    <a
      href="http://creativecommons.org/licenses/by/3.0/"
      target="_blank"
      title="Creative Commons BY 3.0"
    >
      CC 3.0 BY
    </a>
  </div>
</div>
`;
